### PR TITLE
Added COMMAND HELP

### LINKED ISSUE #607: Bug - COMMAND HELP not implemented
```
127.0.0.1:7379> command wrong
(error) ERR unknown subcommand 'WRONG'. Try COMMAND HELP.
127.0.0.1:7379> command help
(error) ERR unknown subcommand 'HELP'. Try COMMAND HELP.
```

I will raise fix PR shortly.

### PR DESCRIPTION
**Pull request by psrvere**
Fixes #607

### CODE REVIEW

File: internal/eval/eval.go [hunk 1]
```diff
@@ -2201,17 +2201,48 @@ func evalCommand(args []string, store *dstore.Store) []byte {
...
+func evalCommandHelp() []byte {
+	format := "COMMAND <subcommand> [<arg> [value] [opt] ...]. Subcommands are:"
+	noTitle := "(no subcommand)"
+	noMessage := "    Return details about all Dice commands."
+	countTitle := "COUNT"
+	countMessage := "    Return the total number of commands in this Dice server."
+	listTitle := "LIST"
+	listMessage := "     Return a list of all commands in this Dice server."
+	getKeysTitle := "GETKEYS <full-command>"
+	getKeysMessage := "     Return the keys from a full Dice command.\r\n"
```
Comment (lucifercr07) [inline internal/eval/eval.go:2227]: nit: `\r\n` can be removed

**Review by lucifercr07 — CHANGES_REQUESTED** [review]
@psrvere thanks for contributing, minor comments.
Shall we add a unit test for this?

File: internal/eval/eval.go (hunk 1 above)
Comment (psrvere) [reply internal/eval/eval.go]: fixed.

**Comment by psrvere** [comment]
@lucifercr07 - added unit test. Please check.

File: internal/eval/eval_test.go [hunk 2]
```diff
@@ -38,42 +38,43 @@ func setupTest(store *dstore.Store) *dstore.Store {
...
-	testEvalHGET(t, store)
-	testEvalPFMERGE(t, store)
-	testEvalJSONSTRLEN(t, store)
-	testEvalJSONOBJLEN(t, store)
-	testEvalHLEN(t, store)
-	testEvalSELECT(t, store)
-	testEvalLLEN(t, store)
-	testEvalGETEX(t, store)
-	testEvalJSONNUMINCRBY(t, store)
+	// testEvalMSET(t, store)
```
Comment (lucifercr07) [inline internal/eval/eval_test.go:41]: @psrvere can you please uncomment these.

Comment (psrvere) [reply internal/eval/eval_test.go]: Apologies. Corrected.

**Pull request merged**